### PR TITLE
Fixed the python implementation of the node

### PR DESCRIPTION
**Pull request by BaBrixx**

### CODE REVIEW

File: common/Message.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class Message:
+    identifier: str
+    timestamp: float
+    signal_strength: float
+
+    def __init__(self, identifier: str, timestamp: float, signal_strength: float):
+        self.identifier = identifier
+        self.timestamp = timestamp
+        self.signal_strength = signal_strength
```
Comment (atjn) [inline common/Message.py]: illegal EOF

Comment (atjn) [inline common/Message.py]:
```suggestion
        self.signal_strength = signal_strength

```

File: common/tracker.py [hunk 2]
```diff
@@ -144,47 +155,49 @@ def register_imsi(self, arfcn, signal_dbm, imsi1="", imsi2="", tmsi1="", tmsi2="
...
+        #if imsi in self.imsistate:
+        #    self.imsistate[imsi]["lastseen"] = now
+        #else:
+        #    self.imsistate[imsi] = {
+        #        "firstseen": now,
+        #        "lastseen": now,
+        #        "imsi": imsi,
+        #        "arfcn": arfcn,
+        #    }
+        #self.imsi_purge_old()
```
Comment (atjn) [inline common/tracker.py]: interesting comment for my brain to parse

Comment (BaBrixx) [reply common/tracker.py]: Yes very

**Review by atjn — APPROVED** [review]
LGTM

**Pull request merged**